syntax = "proto3";

package rpc;

service HeartbeatService {
    rpc SendHeartbeat(stream HeartbeatRequest) returns (HeartbeatResponse);
}

message HeartbeatRequest {
    string client_id = 1;
}

message HeartbeatResponse {
    string status = 1;
}